db/migrate/20210701201156_change_unrecognized_appellant_current_version.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeUnrecognizedAppellantCurrentVersion < ActiveRecord::Migration[5.2]
+  def up
+    UnrecognizedAppellant.where(current_version: nil).where(created_by: nil)
+    .all.each do |unrecognized_appellant|
```

Comment:
I believe this can be simplified into
```suggestion
    UnrecognizedAppellant.where(current_version: nil, created_by: nil)
       .each do |unrecognized_appellant|
```